modulefiles/module-setup.csh.inc

```diff
@@ -2,7 +2,13 @@ set __ms_shell=csh
 
 eval "if ( -d / ) set __ms_shell=tcsh"
 
-if ( { test -d /lfs3 } ) then
+if ( { test -d /lfs/f1 } ) then
```

Comment:
can we try `cat /etc/cluster_name` and then check if it matches `cactus` or `dogwood`?

Reply:
So yes, this works on WCOSS2 and I tested `machine-setup.sh` on Hera/Orion with a change to the WCOSS2 part of the if-block and this works everywhere:
```
if [[ $(cat /etc/cluster_name 2>/dev/null ) == "cactus" || $(cat /etc/cluster_name 2>/dev/null) == "dogwood" ]] ; then
```
^ thoughts on this?

Since this could possibly have unintended consequences at runtime on Hera/Orion without rerunning tests on those platforms (I already tested these PR build changes within the UFS_UTILS tag update testing) and this PR contains only changes that impact WCOSS2 I'd prefer to merge this PR as is and then test this change in a separate PR after testing it at runtime on all three machines. I don't want to break `develop` right before I go on leave. :)

Reply:
```
kate.friedman@clogin05> cat /etc/cluster_name
cactus
kate.friedman@clogin05> git diff machine-setup.sh
diff --git a/sorc/machine-setup.sh b/sorc/machine-setup.sh
index 74c7a705..0ce95e53 100644
--- a/sorc/machine-setup.sh
+++ b/sorc/machine-setup.sh
@@ -20,13 +20,13 @@ target=""
 USERNAME=$(echo $LOGNAME | awk '{ print tolower($0)'})
 ##---------------------------------------------------------------------------
 export hname=$(hostname | cut -c 1,1)
-if [[ -d /lfs/f1 ]] ; then
+if [[ $(cat /etc/cluster_name 2>/dev/null ) == "cactus" || $(cat /etc/cluster_name 2>/dev/null) == "dogwood" ]] ; then
     # We are on NOAA Cactus or Dogwood
     if ( ! eval module help > /dev/null 2>&1 ) ; then
         echo load the module command 1>&2
         source /usr/share/lmod/lmod/init/$__ms_shell
     fi
-    target=wcoss2
+    echo target=wcoss2
     module reset
 
 elif [[ -d /work ]] ; then
kate.friedman@clogin05> sh machine-setup.sh                                                
target=wcoss2
Resetting modules to system default. Reseting $MODULEPATH back to system default. All extra directories will be removed from $MODULEPATH.
```

Reply:
We can use cat later.